R/build-home-authors.R

```diff
@@ -153,7 +153,7 @@ orcid_link <- function(orcid) {
 
   paste0(
     "<a href='https://orcid.org/", orcid, "' target='orcid.widget'>",
-    "<img src='https://members.orcid.org/sites/default/files/vector_iD_icon.svg' class='orcid' alt='ORCID'></a>"
+    "<span class='fab fa-orcid orcid'></span></a>"
```

Comment:
Would you mind adding an `aria-label` to `<a>` and `aria-hidden` to the `<span>` as in https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility#web-fonts-semantic ?

Reply:
Done. I've reused the text from (the now removed) `alt` for `aria-label`. Please let me know if you'd prefer something else.